tests/integration/nm/ovs_test.py

```diff
@@ -582,20 +581,55 @@ def test_ovs_bond_auto_managed_ignored_port(unmaaged_dummy1, eth1_up):
                   - name: {DUMMY1}
                   - name: eth1
         """,
-        Loader=yaml.SafeLoader,
     )
     try:
         libnmstate.apply(desired_state)
         assertlib.assert_state_match(desired_state)
     finally:
         libnmstate.apply(
-            yaml.load(
+            load_yaml(
+                """---
+                interfaces:
+                - name: br0
+                  type: ovs-bridge
+                  state: absent
+                """,
+            )
+        )
+
+
+def test_ovs_port_has_autoconnect_ports(eth1_up):
+    desired_state = load_yaml(
+        """---
+        interfaces:
+        - name: ovs0
+          type: ovs-interface
+        - name: br0
+          type: ovs-bridge
+          state: up
+          bridge:
+            port:
+            - name: ovs0
+            - name: eth1
+        """,
+    )
+    try:
+        libnmstate.apply(desired_state)
+        assert (
+            cmdlib.exec_cmd(
+                "nmcli -g connection.autoconnect-slaves "
+                "c show ovs0-port".split(),
```

Comment:
aha, take me a while to realize why the connection name is `ovs0-port`. (generated by `gen_nm_conn_setting`).